app/Http/Controllers/PostController.php

```diff
@@ -26,13 +26,13 @@ public function index(Request $request): View
         $posts = Post::query()
             ->select('id', 'title', 'is_featured', 'category_id', 'created_at', 'updated_at')
             ->withAggregate('category', 'title')
-            ->when((string) $request->string('search'), function (PostBuilder $query, string $search) {
+            ->when((string) $request->string('search'), function (PostBuilder $query, string $search): void {
```

Comment:
Just wondering, $req->string must always be returning string right? So why extra (string)

Reply:
I think I need to call ->toString() instead 👀